Yoast/YoastSEO.js
# Add markings to keywordDensityAssessment

## Summary

This PR can be summarized in the following changelog entry:

* Adds markings on keyword forms in the text to the assessment that considers density of keyword in the text.

## Relevant technical choices:

* Markings are added to all words within the keyphrase separately. More context on it in #1504.

## Test instructions

This PR can be tested by following these steps:

* Use browserified to check keywordDensityAssessment assessment.
* Single-word or multiple-word keyword: Try using different forms of nouns in the keyword field and in the text. Check if the assessment returns correct result and highlights all relevant forms.

Fixes #1531
